src/server/_namespaces/ts.server.ts

```diff
@@ -1,6 +1,7 @@
 /* Generated file to emulate the ts.server namespace. */
 
 export * from "../../jsTyping/_namespaces/ts.server";
+export * from "../../typingsInstallerCore/_namespaces/ts.server";

```

Comment:
This should be only in tsserverlibrary.js and tsserverlibrary.d.ts and not tsserver.js. 

@jakebailey  is that possible though ?

Reply:
It's possible; just has to be moved to `tsserverlibrary.js`'s `ts.server` file instead.

Reply:
However, it's a little bit weird because `tsserverlibrary` is the API that plugin authors use, and yet the real `tsserver` will not include it, which doesn't sound like it's good either.

Reply:
are you saying people are using tsserverlibrary.d.ts but not tsserverlibrary.js ?

Reply:
Yes, that's everyone who writes a tsserver plugin, effectively.

Reply:
As in, they get passed the ts object via their init method, but that is the view of ts that comes from `src/server`, not the one from `src/tsserver` or `src/tsseverlibrary`, so it's on the whole not accurate, hence why sticking stuff down into `src/server` is the safest move.

Reply:
Is this resolved, then?